File: python/exercise07/area_of_rectanglar_room.py
length = int(input("What is the length of room in feet? "))
width = int(input("What is the width of room in feet? "))
square_meters = (float(length) * float(width) * 0.09290304)

print(f"You entered dimensions of {length} feet by {width} feet.")
print(f"The area is\n{length * width} square feet\n{square_meters:.3f} square meters")

File: javascript/exercise07/area-of-rectanglar-room.js
import { input } from "pynput";

const length = Number(await input("What is the length of room in feet? "));
const width = Number(await input("What is the width of room in feet? "));

const squareMeters = Number(lengtNumber) * Number(width) * 0.09290304;

console.log(`You entered dimensions of ${length} feet by ${width} feet.`);
console.log(
	`The area is\n${length * width} square feet\n${squareMeters.toFixed(3)} square meters`,
);
